lms/static/js/spec/shoppingcart/shoppingcart_spec.js

```diff
@@ -7,7 +7,10 @@ define(['common/js/spec_helpers/ajax_helpers', 'js/shoppingcart/shoppingcart'],
             var requests = null;
 
             beforeEach(function() {
-                setFixtures('<section class="wrapper confirm-enrollment shopping-cart cart-view"><form action="" method="post"><input type="hidden" name="" value="" /><i class="icon fa fa-caret-right"></i><input type="submit" value="Payment"/></form></section>');
+                setFixtures('<section class="wrapper confirm-enrollment shopping-cart cart-view">' +
```

Comment:
If files that should be skipped are linted, we should fix the linter.

> Il giorno 27 mag 2016, alle ore 21:17, Christina Roberts <notifications@github.com> ha scritto:
> 
> In lms/static/js/spec/shoppingcart/shoppingcart_spec.js:
> 
> > @@ -7,7 +7,10 @@ define(['common/js/spec_helpers/ajax_helpers', 'js/shoppingcart/shoppingcart'],
> >              var requests = null;
> >  
> >              beforeEach(function() {
> > -                setFixtures('<section class="wrapper confirm-enrollment shopping-cart cart-view"><form action="" method="post"><input type="hidden" name="" value="" /><i class="icon fa fa-caret-right"></i><input type="submit" value="Payment"/></form></section>');
> > +                setFixtures('<section class="wrapper confirm-enrollment shopping-cart cart-view">' +
> This could also potentially be the cause of the quality error, but given that it is a spec file, I have no desire to do the HTML encoding.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>